package.json

```diff
@@ -32,27 +32,32 @@
     "@angular/platform-browser": "5.2.10",
     "@angular/platform-browser-dynamic": "5.2.10",
     "@ionic-native/core": "4.4.0",
+    "@ionic-native/file": "^4.7.0",
     "@ionic-native/globalization": "^4.7.0",
     "@ionic-native/http": "^4.6.0",
     "@ionic-native/insomnia": "^4.7.0",
+    "@ionic-native/media": "^4.7.0",
     "@ionic-native/screen-orientation": "^4.7.0",
     "@ionic-native/splash-screen": "4.4.0",
     "@ionic-native/status-bar": "4.4.0",
     "@ionic/storage": "2.1.3",
     "angular2-signaturepad": "^2.8.0",
+    "cordova-browser": "5.0.3",
     "cordova-ios": "4.5.4",
-    "cordova-plugin-advanced-http": "^1.11.0",
-    "cordova-plugin-device": "^2.0.1",
+    "cordova-plugin-advanced-http": "^1.11.1",
+    "cordova-plugin-device": "^2.0.2",
     "cordova-plugin-device-authentication":
-      "https://github.com/dvsa/cordova-plugin-device-authentication.git",
+      "git+https://github.com/dvsa/cordova-plugin-device-authentication.git",
```

Comment:
git+ prefix make that app wont build on ionic pro. @danielchildsbjss  Am I right?